src/entry-points/popup/App.svelte

```diff
@@ -400,6 +402,24 @@ along with Jump Cutter Browser Extension.  If not, see <https://www.gnu.org/lice
   $: controllerTypeAlwaysSounded = latestTelemetryRecord?.controllerType === ControllerKind_ALWAYS_SOUNDED;
 
   const displayNewBadgeOnExperimentalAlgorithm = new Date() < new Date('2024-09-30');
+
+  function onAdvancedModeChange() {
+    settingsKeysToSaveToStorage.add('advancedMode');
+    throttledSaveUnsavedSettingsToStorageAndTriggerCallbacks();
+    if (this.checked) {
+      settings.experimentalControllerType = ControllerKind_STRETCHING;
+      settings.marginBefore = 0;
+    }
+  }
+
+  function onSimpleSliderInput() {
+    settingsKeysToSaveToStorage.add('simpleSlider');
+    throttledSaveUnsavedSettingsToStorageAndTriggerCallbacks();
+    settings.volumeThreshold = settings.simpleSlider * 0.001;
+    settings.silenceSpeedRaw = settings.simpleSlider * 0.05 + 1;
+    settings.marginAfter = settings.simpleSlider * 0.01;
```

Comment:
These also need to be added to `settingsKeysToSaveToStorage`, otherwise they don't get saved.